### PR TITLE
chore(data-warehouse): Add field names so they can be filtered

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- can't filter specific data that gets sent to sentry

## Changes

- add custom field names to sensitive information

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Comment by mariusandra** [comment]
Do I understand correctly that the intent is to send the query and the values to sentry upon error, removing the sensitive ones?

If so, I'd suggest adding a method like `add_sensitive_value` onto the context, that prepends `_sensitive` to the returned key, rather than allowing a free form entry field. That should be all we need in this case really, and it'll assign meaning to the keys, instead of randomness.

**Comment by EDsCODE** [comment]
yep good call

File: posthog/hogql/context.py [hunk 1]
```diff
@@ -38,7 +38,15 @@ class HogQLContext:
...
         self.values[key] = value
+        return key
+
+    def add_value(self, value: Any) -> str:
+        key = self._add_value(value)
         return f"%({key})s"
+
+    def add_sensitive_value(self, value: Any) -> str:
+        key = self._add_value(value)
+        return f"%(sensitive_{key})s"
```
Comment (mariusandra) [inline posthog/hogql/context.py:52]: Keep it simple:

```suggestion
    def add_value(self, value: Any) -> str:
        key = f"hogql_val_{len(self.values)}"
        self.values[key] = value
        return key

    def add_sensitive_value(self, value: Any) -> str:
        key = f"hogql_val_{len(self.values)}_sensitive"
        self.values[key] = value
        return f"%({key})s"
```

**Review by mariusandra — APPROVED** [review]
👍

**Pull request merged**